scripts/create_pelorus_operator.sh

```diff
@@ -130,7 +130,11 @@ pushd "${destination_dir}" || exit 2
     echo "INFO: Creating operator init files"
     operator-sdk init --plugins=helm --domain "${OPERATOR_PROJECT_DOMAIN}" --project-name "${OPERATOR_PROJECT_NAME}" || exit 2
     echo "INFO: Creating api"
+    # https://github.com/operator-framework/operator-sdk/issues/4689#issuecomment-992967172
+    mkdir -p helm-charts/pelorus/charts
+    cp -r "${source_dir}charts/exporters" helm-charts/pelorus/charts
     operator-sdk create api --helm-chart="${source_dir}" || exit 2
```

Comment:
To test what you want @mpryc , I think you just need to remove lines 134 and 135 and change here (line 136) to `operator-sdk create api --helm-chart="${source_dir}charts/exporters/" || exit 2`